contracts/mocks/token/ERC7984FreezableMock.sol

```diff
@@ -26,7 +25,9 @@ contract ERC7984FreezableMock is ERC7984Freezable, AccessControl, HandleAccessMa
     }
 
     function confidentialAvailableAccess(address account) public {
-        getHandleAllowance(euint64.unwrap(confidentialAvailable(account)), account, true);
+        euint64 available = confidentialAvailable(account);
+        FHE.allowThis(available);
```

Comment:
This still isn't necessary given that this contract isn't storing the `available` handle

Reply:
ah yes the contract needs to have access for offchain decryption to work. Forgot about that